config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/studios", to: "studios#index"
+
+  get "/movies/:id", to: "movies#show"
+  
+  post "/movies/:id/casts", to: "casts/movies#create"
```

Comment:
This path makes sense but see note in cast controller about convention for namespacing like this